ghost/email-service/lib/email-event-processor.js

```diff
@@ -82,10 +82,8 @@ class EmailEventProcessor {
                 emailId: recipient.emailId,
                 timestamp
             });
-            await this.#eventStorage.handleOpened(event);
-
             this.#domainEvents.dispatch(event);
-            await waitForEvent(); // Avoids knex connection pool to run dry
```

Comment:
The waitForEvent delayed processing a batch of 300 events from 1.5s to 20s locally.